addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitTaggedItem.java

```diff
@@ -36,9 +37,13 @@
     public HomekitTaggedItem(Item item, ItemRegistry itemRegistry) {
         this.item = item;
         for (String tag : item.getTags()) {
-            if (item instanceof DimmerItem) {
+
+            if (item instanceof ColorItem) {
+                tag = "Colorful" + tag;
```

Comment:
I know you're just extending the existing pattern here, but that was only there for legacy reasons. Before we had an ESH ontology, users tagged dimmable items separately.

I think it would make more sense to handle the check for DimmerItem and ColorItem in a HomekitLightingFactory class and use that in HomekitAccessoryFactory